homeassistant/components/mqtt/scene.py

```diff
@@ -19,22 +19,22 @@
 from .mixins import (
     CONF_OBJECT_ID,
     MQTT_AVAILABILITY_SCHEMA,
-    MqttAvailability,
-    MqttDiscoveryUpdate,
+    MqttEntity,
     async_setup_entry_helper,
     async_setup_platform_helper,
-    init_entity_id_from_config,
 )
 
 DEFAULT_NAME = "MQTT Scene"
 DEFAULT_RETAIN = False
 
+DEFAULT_PAYLOAD_ON = "ON"
+
 PLATFORM_SCHEMA = mqtt.MQTT_BASE_PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_COMMAND_TOPIC): mqtt.valid_publish_topic,
         vol.Optional(CONF_ICON): cv.icon,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PAYLOAD_ON): cv.string,
+        vol.Optional(CONF_PAYLOAD_ON, default=DEFAULT_PAYLOAD_ON): cv.string,
```

Comment:
This is in the documentation, but the default is was not set.

Reply:
This is a breaking change, let's update the documentation according to the implementation instead.

Reply:
already meregd

Reply:
I'll open a new PR to re-add it

Reply:
I'll open a PR to change the current docs that a default value is not set.

Reply:
OK, let's revert this change then.